custom/icds_reports/tests/agg_tests/test_export_data.py

```diff
@@ -336,20 +336,20 @@ def test_children_export_child_six(self):
             data,
             [
                 'st2',
-                732, 1039, '70.45 %',
-                30, 1003, '2.99 %',
+                730, 1037, '70.40 %',
+                30, 1001, '3.00 %',
                 307,
-                18, 732, '2.46 %',
-                138, 732, '18.85 %',
-                576, 732, '78.69 %',
+                18, 730, '2.47 %',
+                136, 730, '18.63 %',
+                576, 730, '78.90 %',
                 1, 23, '4.35 %',
                 4, 23, '17.39 %',
                 18, 23, '78.26 %',
                 11, 32, '34.38 %',
                 6, 32, '18.75 %',
                 15, 32, '46.88 %',
                 0, 4, '0.00 %',
-                87, 1218, '7.14%',
+                87, 1216, '7.15%',
```

Comment:
why is this changed by factor of 2 not 1? @challabeehyv

Reply:
we have changed two months fixtures. That is why the aggregation shows two missing